packages/radio-group/test/radio-button.test.js

```diff
@@ -1,7 +1,6 @@
 import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
-import { fixtureSync, nextFrame, space, spaceKeyDown, spaceKeyUp, touchstart, touchend } from '@vaadin/testing-helpers';
-import { down, up } from './helpers.js';
+import { fixtureSync, nextFrame, space, touchstart, touchend, fire } from '@vaadin/testing-helpers';
```

Comment:
Have you tried to import `down` and `up` from `@vaadin/testing-helpers`? It exports those helpers.

Reply:
Yeah, but when I use them, tests break for some reason. Created the https://github.com/vaadin/web-components/issues/2425 issue.

```
packages/radio-group/test/radio-button.test.js:

 ❌ radio-button > click method > should be called on mouseup
      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at o.<anonymous> (packages/radio-group/test/radio-button.test.js:167:35)

 ❌ radio-button > change event > should fire on mouseup
      AssertionError: expected false to be true
      + expected - actual

      -false
      +true
```